### PR TITLE
Comment counter working again

### LINKED ISSUE #74: Comments are not counted per thread
Apparently the server side counting is broken

### PR DESCRIPTION
**Pull request by faboweb**
fixes #74

### CODE REVIEW

File: functions/countComments.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
-const functions = require('firebase-functions');
-const admin = require('firebase-admin');
+const functions = require('firebase-functions')
+const admin = require('firebase-admin')
 
 exports.createComment = functions.database.ref('/comments/{threadId}/{commentId}').onCreate((event) => {
   getRessource(event.params.threadId, ressource => {
     const parentRef = admin.database().ref('/' + ressource + '/' + event.params.threadId)
```
Comment (jolesbi) [inline functions/countComments.js]: is `ressource` a typo here?

File: functions/countComments.js [hunk 2]
```diff
@@ -1,14 +1,15 @@
-const functions = require('firebase-functions');
-const admin = require('firebase-admin');
+const functions = require('firebase-functions')
+const admin = require('firebase-admin')
 
 exports.createComment = functions.database.ref('/comments/{threadId}/{commentId}').onCreate((event) => {
   getRessource(event.params.threadId, ressource => {
```
Comment (jolesbi) [inline functions/countComments.js]: general question: does `ressource` have two `ss`'s on purpose?

Comment (faboweb) [reply functions/countComments.js]: no... *blush
will change it! :D

Comment (faboweb) [reply functions/countComments.js]: done in #78

**Pull request merged**